lib/fake_async.dart

```diff
@@ -272,13 +276,23 @@ class _FakeTimer implements Timer {
   /// fired.
   Duration _nextCall;
 
+  /// The current stack trace when this timer was created.
+  final StackTrace _creationStackTrace;
```

Comment:
You could inline the initialization

final _creationStackTrace = StackTrace.current;

Reply:
Ah sure, I was just keeping the implementations identical but might as well.